src/telliot_feeds/reporters/tellorflex.py

```diff
@@ -217,46 +210,58 @@ async def get_num_reports_by_id(self, query_id: bytes) -> Tuple[int, ResponseSta
         count, read_status = await self.oracle.read(func_name="getNewValueCountbyQueryId", _queryId=query_id)
         return count, read_status
 
-    async def fetch_datafeed(self) -> Optional[DataFeed[Any]]:
+    async def rewards(self) -> int:
         tip = 0
-
-        if self.datafeed is not None:
-            # Skip fetching datafeed & checking profitability
-            if self.expected_profit == "YOLO":
-                return self.datafeed
-
-            single_tip, status = await self.autopay.get_current_tip(self.datafeed.query.query_id)
-            if not status.ok:
-                logger.warning("Unable to fetch single tip")
-                return None
-
+        datafeed: DataFeed[Any] = self.datafeed  # type: ignore
```

Comment:
You can get rid of this line and just use `self.datafeed` elsewhere, no?

Reply:
I was trying to bypass tox typing in more lines by doing it once here